src/VoyagerServiceProvider.php

```diff
@@ -155,15 +155,25 @@ protected function addStorageSymlinkAlert()
 
         $storage_disk = (!empty(config('voyager.storage.disk'))) ? config('voyager.storage.disk') : 'public';
 
-        if (request()->has('fix-missing-storage-symlink') && !file_exists(public_path('storage'))) {
-            $this->fixMissingStorageSymlink();
-        } elseif (!file_exists(public_path('storage')) && $storage_disk == 'public') {
-            $alert = (new Alert('missing-storage-symlink', 'warning'))
-                ->title(__('voyager.error.symlink_missing_title'))
-                ->text(__('voyager.error.symlink_missing_text'))
-                ->button(__('voyager.error.symlink_missing_button'), '?fix-missing-storage-symlink=1');
-
-            VoyagerFacade::addAlert($alert);
+        if (request()->has('fix-missing-storage-symlink')) {
+            if (file_exists(public_path('storage'))) {
+                if (readlink(public_path('storage')) == public_path('storage')) {
+                    rename(public_path('storage'), 'storage_old');
```

Comment:
> [`rename()`](http://php.net/manual/en/function.rename.php): If renaming a directory and newname exists, this function will emit a warning.

Can you add `date('Y-m-d H-i-s')` after `old`?

Reply:
Normally we tell users to delete this directory.
I think 99% of the users who have this problem dont even know/care that they have this folder